tests/durabletask-azuremanaged/test_dts_orchestration_versioning_e2e.py

```diff
@@ -0,0 +1,319 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import json
+import os
+
+import pytest
+
+from durabletask import client, task, worker
+from durabletask.azuremanaged.client import DurableTaskSchedulerClient
+from durabletask.azuremanaged.worker import DurableTaskSchedulerWorker
+
+# NOTE: These tests assume a sidecar process is running. Example command:
```

Comment:
This comment probably doesn't make sense - but maybe upate it with instructions to spin up the DTS emulator and run against that instead?

Reply:
Done - updated all the e2e instructions, including the durabletask ones pointing to durabletask-go instead of chris's sidecar fork